crates/modelardb_server/src/storage/compressed_data_manager.rs

```diff
@@ -43,19 +42,14 @@ pub(super) struct CompressedDataManager {
     /// The compressed segments before they are saved to persistent storage. The key is the name of
     /// the model table the compressed segments represents data points for so the Apache Parquet
     /// files can be partitioned by table.
-    compressed_data_buffers: DashMap<String, CompressedDataBuffer>,
+    pub(super) compressed_data_buffers: DashMap<String, CompressedDataBuffer>,
```

Comment:
Unclear why `compressed_data_buffer` has become `pub(super)` after the removal of the metrics. Maybe document it in an answer to this comment so we keep that knowledge.